app/javascript/packs/components/RangeOption.jsx

```diff
@@ -57,7 +57,7 @@ class RangeOption extends React.Component {
       >
         <ContentEditable
           onChange={() => 0}
-          onKeyDown={(e) => this.onChange(e.target.innerText)}
+          onKeyDown={(e, val) => this.onChange(val)}
```

Comment:
Hmm, for some reason `onKeyDown` doesn't work as expected -- `val` should be the number that was typed in the range option, but it's always undefined. Right now typing in a range option sets the answer text to undefined and breaks everything. Can you check if this is the right thing to use?